index.html

```diff
@@ -2636,12 +2636,12 @@ <h2 id="docconformance">
             </td>
             <td>
               <p>
-                <code>role=<a href="#index-aria-cell">cell</a></code> if a
-                descendant of a `table` element.
+                <code>role=<a href="#index-aria-cell">cell</a></code> if the ancestor
+                `table` element has `role=table`.
               </p>
               <p>
-                <code>role=<a href="#index-aria-gridcell">gridcell</a></code> if a
-                descendant of a `table` element with `role=grid` or `treegrid`.
+                <code>role=<a href="#index-aria-gridcell">gridcell</a></code> if the ancestor
+                `table` element has `role=grid` or `treegrid`.
```

Comment:
```suggestion
                `table` element is exposed as a `role=grid` or `treegrid`.
```

Reply:
changed this for consistency, but could well see this instance staying "has", since a table would not be exposed as these roles without the explicit declaration.  I could go either way here.